Modules/IO/ImageBase/include/itkImageSeriesReader.hxx

```diff
@@ -436,17 +437,15 @@ ImageSeriesReader<TOutputImage>::GenerateData()
               outputSpacing[this->m_NumberOfDimensionsInImage])) // either non-uniform sampling or missing slice
         {
           nonUniformSampling = true;
+          spacingDeviation = Math::abs(outputSpacing[this->m_NumberOfDimensionsInImage] - dirNnorm);
           itkWarningMacro(<< "Non unform sampling or missing slices detected , expected "
-                          << outputSpacing[this->m_NumberOfDimensionsInImage] << " got: " << dirNnorm);
-
-          EncapsulateMetaData<bool>(output->GetMetaDataDictionary(),
-                                    "ITK_non_uniform_sampling",
-                                    true); // set metadata for the series reader output
+                          << outputSpacing[this->m_NumberOfDimensionsInImage] << " got: " << dirNnorm
+                          << " Deviation of:" << spacingDeviation);
```

Comment:
With some of our datasets this logs "Non unform sampling or missing slices detected , expected 6 got: 6".  Can we use std::setprecision() or something similar to get more digits?

Also, just noticed typo: unform -> uniform